deltacat/compute/metastats/stats.py

```diff
@@ -94,7 +94,11 @@ def start_stats_collection(
 
 
 def _get_account_id() -> str:
-    client = client_cache("sts", None)
+    client = client_cache(
+        "sts",
+        region_name="us-east-1",
+        endpoint_url="https://sts.us-east-1.amazonaws.com",
```

Comment:
What would happen if the environment is not setup in us-east-1? The region and endpoint must be picked from environment variable.

Reply:
The most recent change will first check the environment variable before setting the region and endpoint url. In the event that a node is not in us-east-1 and does not have the environment variable set, [they will still be able to use the us-east-1 endpoint](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html), although just at higher latency.